### DISCUSSION TITLE
Real usecase for watchtower?

### DISCUSSION THREAD

**Discussion by ArjunDandagi** [containrrr/watchtower#1330] [Q&A] [answered]
Hello folks, 
I came across this  by knowing from a label that was used in [drone](https://github.com/harness/drone/blob/master/docker/Dockerfile.agent.linux.amd64#L16)

i would like to know if there is a real power level use case of watchtower, where is it being used extensively and why watchtower, 
the motivation, the pain point that you felt for creating the software. 

Thanks,
Arjun

**Comment by piksel** [comment]
The current maintainers were not the ones who initially created it, so we can't really tell you much about its conception. As to its usage, I would estimate that it's generally used in "home lab" type of scenarios. I'm not sure what "power level" that is though 😅

**Comment by simonaronsson** [comment, marked as answer]
The vast majority of users are using Watchtower in either a home lab or a media center setup. We did a poll a couple of years ago to get a feel for the community mix, and while the prior two were by far the most common ones, there were quite a few production deployments as well. By our estimates the usage is quite widespread, which also was confirmed at some point by the Docker Hub team.  

It does what it says and not much more: automatically recreates containers whenever there is an update to the published image. For that, it works flawlessly most of the time. Whether that is "power level" enough for you is completely up to you to decide. 😅

**Comment by ArjunDandagi** [comment]
Thanks for helping me understanding the use cases and application @piksel  and @simskij
